mod/engine-primitives/pkg/engine-primitives/attributes.go

```diff
@@ -63,18 +63,16 @@ type PayloadAttributes[
 	ParentBeaconBlockRoot common.Root `json:"parentBeaconBlockRoot"`
 }
 
-// NewPayloadAttributes creates a new PayloadAttributes.
-func NewPayloadAttributes[
```

Comment:
This is not really needed and used only in tests. The `New` method is required by a few interfaces